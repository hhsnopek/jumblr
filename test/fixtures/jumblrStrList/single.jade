include ../../../lib/jumblr

+jumblrStrList({'VideoThumbnail': 'video'})
div(class= jumblrStrList) {VideoThumbnailURL}
