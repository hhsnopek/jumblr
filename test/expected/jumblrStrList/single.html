<div class="{block:VideoThumbnail}video{/block:VideoThumbnail}">{VideoThumbnailURL}</div>
